back/spec/models/custom_field_spec.rb

```diff
@@ -386,6 +386,14 @@ def visit_ranking(_field)
       # 'somewhere else' option should be the last.
       expect(domicile_field.options.last.area).to be_nil
     end
+
+    describe 'ordered_options' do
+      it 'returns the ordered domicile options with keys and titles from areas' do
+        options = domicile_field.ordered_options
+        expect(options.pluck(:key)).to eq(Area.pluck(:id) + ['outside'])
+        expect(options.last.title_multiloc['en']).to eq 'Somewhere else'
```

Comment:
I find it surprising that this works, because in the fixtures I see "somewhere else" with small "s", which shouldn't be equal.